### PR TITLE
Changed _dev-only to dev-only

### LINKED ISSUE #474: The `yarn dev-only` misspelled in `01_getting_started.md`
## Product

In the **Design System** getting started docs `01_getting_started.md` the command `yarn dev-only` is misspelled. It should be `yarn _dev-only`

## Additional information

In the docs
image
In the package.json
image

### PR DESCRIPTION
**Pull request by BabyElias**
## Description
Changed command name from _dev-only to dev-only for easiness.

#### Issue addressed
Fixes #474

### CODE REVIEW

File: package.json
Comment (akolson) [inline package.json:15]: Please note that updating the above would cause the `dev` script to break. Please update `dev` script to 
`"dev": "npm-run-all --parallel dev-only _lint-watch-fix _api-watch pregenerate",`. (i.e. replace the usages of `_dev-only` with `dev-only`)

**Review by akolson — CHANGES_REQUESTED** [review]
Hi @BabyElias. Great job! I have left a comment for you to address. 

Also, please be sure to update the CHANGLOG.md file with required information as stipulated in the guidelines. This same information(changelog) is also relevant for your PR description(See the [template here](https://github.com/learningequality/kolibri-design-system/blob/main/.github/pull_request_template.md#changelog))

Please be sure to let us know incase you have an questions or issues to clarify. Thanks

**Comment by BabyElias** [comment]
Done with the required changes :)

File: CHANGELOG.md
Comment (akolson) [inline CHANGELOG.md:12]: @BabyElias  Could you please change this to `Script enhancement` and not `Bug fix`.

Comment (BabyElias) [reply CHANGELOG.md]: Done.

**Review by akolson — APPROVED** [review]
These changes look correct to me. Thanks @BabyElias for your contribution in making the Kolibri ecosystem better.  cc @MisRob @thanksameeelian

File: CHANGELOG.md
Comment (MisRob) [inline CHANGELOG.md:18]: Could you also please paste the link to the PR here on line 19, like it's done for other changelog items?

`[#478]: https://github.com/learningequality/kolibri-design-system/pull/478`

This will ensure that [#478] on line 10 will work as intended.

---

Since you volunteered for another issue - when opening a PR next time, I'd recommend not removing the changelog item from a pull request description and fill it in there. Then, we can just paste it to the changelog.md before merging. This should help to keep it in the right format.

We realize that for some types of work, a core team member will need to fill it out. So generally, any time you don't know what to write in the pull request, you can just keep some changelog fields empty, let us know about it and we will take care of it :)

**Comment by MisRob** [comment]
Congratulation to your first contribution, @BabyElias!

File: CHANGELOG.md
Comment (BabyElias) [reply CHANGELOG.md:20]: Sure! I'll keep these things in mind next time. 
Thank you for your guidance and patience.

Comment (MisRob) [reply CHANGELOG.md:20]: Sure, no worries! This is a new process to make creation of release notes easier and we're still figuring things out, especially when it comes to contributions. If you had any feedback or ideas on how to make this straightforward, we're happy to hear back from you.

**Comment by BabyElias** [comment]
Any more changes that are needed?

**Comment by akolson** [comment]
Hi @BabyElias! I don't think so. We should be good to go now and this should be merged soon. Thanks

**Pull request merged**
